From 63bdd9e26d72427008c4a63ea371d0da0329947f Mon Sep 17 00:00:00 2001
Message-Id: <63bdd9e26d72427008c4a63ea371d0da0329947f@dist-git>
From: John Ferlan <jferlan@redhat.com>
Date: Thu, 9 Jul 2015 08:28:50 -0400
Subject: [PATCH] docs: Clarify unprivileged sgio feature

https://bugzilla.redhat.com/show_bug.cgi?id=1072736

Update the descriptions for disk and hostdev sgio in order to indicate
not all hypervisors and OS's support this feature

Signed-off-by: John Ferlan <jferlan@redhat.com>
(cherry picked from commit 4f4ac3913a379ad7d3880c57cfe7304c4a22cc22)
Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
---
 docs/formatdomain.html.in | 19 ++++++++++---------
 1 file changed, 10 insertions(+), 9 deletions(-)

diff --git a/docs/formatdomain.html.in b/docs/formatdomain.html.in
index e2ccfa5..cb11b1f 100644
--- a/docs/formatdomain.html.in
+++ b/docs/formatdomain.html.in
@@ -2004,10 +2004,10 @@
           <dt><code>sgio</code> attribute
           <span class="since">since 1.0.2</span></dt>
             <dd>
-            Indicates whether the kernel will filter unprivileged
-            SG_IO commands for the disk, valid settings are "filtered" or
-            "unfiltered". Defaults to "filtered". Similar to <code>rawio</code>,
-            <code>sgio</code> is only valid for device 'lun'.
+            If supported by the hypervisor and OS, indicates whether
+            unprivileged SG_IO commands are filtered for the disk. Valid
+            settings are "filtered" or "unfiltered" where the default is
+            "filtered". Only available when the <code>device</code> is 'lun'.
             </dd>
           <dt><code>snapshot</code> attribute
           <span class="since">since 0.9.5</span></dt>
@@ -3239,11 +3239,12 @@
           </dd>
           <dt>scsi</dt>
           <dd>For SCSI devices, user is responsible to make sure the device
-            is not used by host. The optional <code>sgio</code>
-            (<span class="since">since 1.0.6</span>) attribute indicates
-            whether the kernel will filter unprivileged SG_IO commands for
-            the disk, valid settings are "filtered" or "unfiltered".
-            The default is "filtered". The optional <code>rawio</code>
+            is not used by host. If supported by the hypervisor and OS, the
+            optional <code>sgio</code> (<span class="since">since 1.0.6</span>)
+            attribute indicates whether unprivileged SG_IO commands are
+            filtered for the disk. Valid settings are "filtered" or
+            "unfiltered", where the default is "filtered".
+            The optional <code>rawio</code>
             (<span class="since">since 1.2.9</span>) attribute indicates
             whether the lun needs the rawio capability. Valid settings are
             "yes" or "no". See the rawio description within the
-- 
2.5.1

